docs/intro-react-native-components.md

```diff
@@ -0,0 +1,100 @@
+---
+id: intro-react-native-components
+title: Core Components and Native Components
+---
+
+React Native is an open source framework for building Android and iOS applications using [React](https://reactjs.org/) and the app platform’s native capabilities. With React Native, you use JavaScript to access your platform’s APIs as well as to describe the appearance and behaviour of your UI using React components, bundles of reusable, nestable code. You can learn more about React in the next section. But first, let's cover how components work in React Native.
```

Comment:
Nit: Use a colon instead of comma: "UI using React components: bundles of reusable, nestable code"

Reply:
A dash would work too. I thought this was a run-on sentence at first.